packages/bootstrap-4/src/RadioWidget/RadioWidget.tsx

```diff
@@ -22,45 +19,43 @@ const RadioWidget = ({
 }: WidgetProps) => {
   const { enumOptions, enumDisabled } = options;
 
-  const _onChange = ({}, value: any) =>
+  const _onChange = ({
+    target: { value },
+  }: React.ChangeEvent<HTMLInputElement>) =>
     onChange(schema.type == "boolean" ? value !== "false" : value);
   const _onBlur = ({ target: { value } }: React.FocusEvent<HTMLInputElement>) =>
     onBlur(id, value);
   const _onFocus = ({
     target: { value },
   }: React.FocusEvent<HTMLInputElement>) => onFocus(id, value);
 
-  const row = options ? options.inline : false;
+  const inline = options.inline !== null ? options.inline : false;
 
   return (
-    <>
-      <FormLabel required={required} htmlFor={id}>
-        {label || schema.title}
-      </FormLabel>
-      <RadioGroup
-        value={`${value}`}
-        row={row as boolean}
-        onChange={_onChange}
-        onBlur={_onBlur}
-        onFocus={_onFocus}>
-        {(enumOptions as any).map((option: any, i: number) => {
-          const itemDisabled =
-            enumDisabled && (enumDisabled as any).indexOf(option.value) != -1;
-
-          const radio = (
-            <FormControlLabel
-              control={<Radio color="primary" key={i} />}
-              label={`${option.label}`}
-              value={`${option.value}`}
-              key={i}
-              disabled={disabled || itemDisabled || readonly}
-            />
-          );
-
-          return radio;
-        })}
-      </RadioGroup>
-    </>
+    <Form.Group controlId={id} className="mb-0">
+      <Form.Label className="d-block">{label || schema.title}</Form.Label>
+      {(enumOptions as any).map((option: any, i: number) => {
+        const itemDisabled =
+          enumDisabled && (enumDisabled as any).indexOf(option.value) != -1;
+        const checked = option.value === value;
+        const radio = (
+          <Form.Check
+            inline={inline as boolean}
+            label={`${option.label}`}
```

Comment:
I know this is inherited from the old package, but I don't think string coercion makes sense here. `Form.Check.label` prop type is `ReactNode`, so the value could be:
- `null`, which would be coerced to string "null"
- `ReactElement`, which would be coerced to some kind of object string representation?

I'd just pass it through:
```suggestion
            label={option.label}
```